# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-rotated-clip.html.ini
# Commit: 9fa6f71111d0
# Full Hash: 9fa6f71111d0d77477abc4502b721d8d4ba23604
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 336a2aa0a29e -r 9fa6f71111d0 testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-rotated-clip.html.ini
--- a/testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-rotated-clip.html.ini	Sat Feb 19 23:27:02 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-[mix-blend-mode-rotated-clip.html]
-  expected:
-    FAIL